### ISSUE TITLE
Cannot resize images anymore

### ISSUE THREAD

**Issue by AlbertRapp** [quarto-ext/lightbox#5]
After including lightbox in my Quarto blog, chunk options like `fig-width` or `out-width` do not work anymore. Images are always included in full-size.

Even regular Markdown like `![](img.png){fig-align="center" width="70%"}` does not react anymore. When I remove lightbox from `_quarto.yml`, everything works as expected again.

I've just run `quarto update extension quarto-ext/lightbox` on my blog project but the problem persists. My current workaround is to create a CSS grid manually.

```
::: {.grid}

::: {.g-col-6 .g-start-4}
![](img.png)
:::

:::
```

**Comment by dragonstyle** [comment]
Thank you for reporting this! This should be fixed in version `0.1.1` - run `quarto update extension quarto-ext/lightbox` to install the latest version.

**Comment by AlbertRapp** [comment]
Works perfectly. Thank you!
